dynamic_rest/prefetch.py

```diff
@@ -205,13 +239,24 @@ def execute(self):
         # TODO: check if queryset already has values() called
         # TODO: use self.fields
         qs = self.queryset._clone()
-        data = list(qs.values())
 
-        self.merge_prefetch(data)
+        has_fastquery = hasattr(self.model, 'use_fastquery')
+        use_models = has_fastquery and self.model.use_fastquery == False
+
+        if use_models:
+            self._data = FastList(
+                map(lambda obj: SlowObject(
+                    obj, pk_field=self.pk_field
+                ), qs.all())
```

Comment:
If there are downstream prefetches, this queryset doesn't have those. It might be possible to here get `self.prefetches`, and pass to `qs.prefetch_related()`.